DMCompiler/Compiler/DM/DMASTHelper.cs

```diff
@@ -5,11 +5,11 @@
 using System.Reflection;
 using System.Text;
 
-namespace OpenDreamShared.Compiler.DM {
+namespace DMCompiler.Compiler.DM {
 
 
     public static partial class DMAST {
-        private static List<Type> printable_field_types = new() { typeof(string), typeof(Dream.DreamPath), typeof(int), typeof(float) };
+        private static List<Type> printable_field_types = new() { typeof(string), typeof(int), typeof(float) };
```

Comment:
You don't need to remove DreamPath from this, it can be accessed from the `OpenDreamShared.Dream` namespace.